bindgen/codegen/mod.rs

```diff
@@ -4358,7 +4364,7 @@ impl TryToRustTy for Type {
                 utils::build_path(item, ctx)
             }
             TypeKind::Opaque => self.try_to_opaque(ctx, item),
-            TypeKind::Pointer(inner) | TypeKind::Reference(inner) => {
+            TypeKind::Pointer(inner) | TypeKind::Reference(inner, _) => {
```

Comment:
Do we handle `Pointer` and `Reference` meaningfully differently somewhere else? I wonder if instead of adding `Reference(..., ReferenceKind)` we should just have `Pointer(inner, PointerKind)` where `PointerKind { Regular, LValueReference, RValueReference }` or so.